.travis.yml

```diff
@@ -1,10 +1,20 @@
 sudo: required
-services:
-  - docker
 
-before_install:
-- docker pull edbrown23/sysadmin:1
-- docker build . -t sysadmin_tester
+env:
+  global:
+  - secure: Nk0lL86bdT9HlUzgFGz3XCckTdss0cjZY9mzbNh2XttyMxRws4ar62PNuYHHgM8h4NIuIzdiCC+ulrTVQgqx4ui/2SxkAUgFrSKlYrdfKEIYgTEa/RkRCYrLs40OEqOgxKDhwudi6c3Vx5J3dWBllrFKIU4C3heOJIibVmvQrarZojQc5bAimKFAXl9ZobezD3VtkW+az9lSzYN93ypdo3yAflTPPWfvF/rDVTz2avJsExtPv43f+J1mQj+AiiU4L0qXkHwBV+a+A/ZqHjmreSyUmShFXX7bn9qcdb6SKQBbS/i/qGahUVrbJ4lPkpjAHEmZ8fi6yz8nj3GoahcA9sQ6NgpLtSPhy47GLeDwdMoOTq/lrwKPPAwPTUjrGeM/VfKBFiWa12J30EMH5Nv2EzpNlY3bEodRIQSiN02G6xzfyK7XDyg4nUMraIPwCs3lAv4wEl3LIImMkyOnQexE0oFkCEAOMHde0VszTUphPnNDLMY6VaMls1JVzTTgWO8e1GGQVB4lISPDLehTMLnJZE43zrHrhEf7oA3O6YtRezpuB+7wJ3YfNsPQh5FQ7GMAhAufH5ss/Ql05D8WfCjUmaFkbagixcUozaNKfct6MFgWRUw5fdXAd37A6DoCeRXoO9vH7nTDm443aG2uZYavdGGjNCPTnq2atOjHj6itRbc=
```

Comment:
Should these be here? If they're meant to be env vars, that's possible within travis

Reply:
It's not possible for me to set env vars in travis. That was what I tried to do first and it wouldn't allow me. I couldn't find info on why I wasn't allowed. It might be because I didn't make the travis project.

As far as I can tell this is basically the same as setting env vars. https://docs.travis-ci.com/user/encryption-keys/ It won't work on PR's from forks, but we won't be pushing new images from forks so It probably doesn't matter.